### DISCUSSION TITLE
POI Finder

### DISCUSSION THREAD

**Discussion by berend** [craigins/CoreKeeperMapTool#8] [Ideas]
I was wondering, if it is possible to have a fixed set of special blocks and search the uploaded map for that. And then put some markers on the map.

Examples:
* legendary bow locatrions like the vault and the bigger cities
* larva location
* mazes

As a start, one could show all indestructable tiles, that should cover the bow vault, ancient pickaxe location and that one part from the legendary sword.

I am not a JS guy, but I would like to help here

**Comment by berend** [comment]
oh, jsut checked, highlighting obsidian wall is already possible ...

**Comment by craigins** [comment]
The indestructible tiles are called Obsidian, and you can filter to those tiles using the filter on the left.

One issue is the map only shows what you have uncovered, it doesn't have access to the world file (which is encrypted) and even then, specific scenes like Larva and mazes are generated at random as you explore, they aren't predefined at world creation.

We could look into using some sort of image recognition to highlight previously found unique scenes.  But if you've already uncovered them on the map, it won't be of too much use.

**Reply by berend** [reply]
> Larva and mazes are generated at random as you explore
Ah, ok. I didnt know.

I saw that there was a world seed (in the decicated server settings). So, even for the same world seed, the map can be different each time?

**Reply by craigins** [reply]
seeds typically initialize a random number generator.  While you could use the same seed for a new world, it would likely only duplicate things like boss positions and biome locations.  any user action beyond that would cause the randomness to deviate between worlds.
